spec/pipelines/pacbio/sample_sheet_spec.rb

```diff
@@ -57,19 +57,88 @@
       expect(column.fetch('populate')).to eq(populate)
     end
 
-    it 'will have a plate 1' do
+    it 'will have a plate-1' do
       column = sample_sheet_configuration.columns.children['Plate 1']
       expect(column).to be_present
       expect(column.fetch('type')).to eq(:model)
-      expect(column.fetch('value')).to eq('plate.sequencing_kit_box_barcode')
+      expect(column.fetch('value')).to eq('sequencing_kit_box_barcode_plate_1')
       expect(column.fetch('populate')).to eq(populate)
     end
 
-    it 'will have a plate 2' do
+    it 'will not have a plate-2' do
       column = sample_sheet_configuration.columns.children['Plate 2']
       expect(column).to be_present
+      expect(column.fetch('type')).to eq(:model)
+      expect(column.fetch('value')).to eq('sequencing_kit_box_barcode_plate_2')
+      expect(column.fetch('populate')).to eq(populate)
+    end
+
+    # Where should the rest of the sample sheet columns be tested?
```

Comment:
Remove comment. We do some testing in spec/csv_generator but not sure how that will fit in with https://github.com/sanger/traction-service/pull/1057/files#diff-58f290e783dc1f04c7c476eb312d72edd873a2c4c64f9e27b6c8693548b854d9.